### PR TITLE
Ignore more development files from npm package

### PR DESCRIPTION
**Pull request by ai**
Make `node_modules` smaller.

### CODE REVIEW

**Review by giuseppeg — COMMENTED** [review]
should we remove .npmignore altogether and add `files` to package.json?

**Comment by ai** [comment]
Sure. Will do it a little later.

**Comment by ai** [comment]
> should we remove .npmignore altogether and add files to package.json?

Done

File: package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
   },
   "scripts": {
     "test": "ava test/test.js && eslint *.js test/*.js"
-  }
+  },
+  "files": [
+    "index.js"
```
Comment (giuseppeg) [inline package.json:34]: do we need to add the LICENSE file too?

Comment (ai) [reply package.json:34]: `package.json`, `LICENSE`, and `README.md` is by default and `files` do not affect them

**Pull request merged**